PowerShell/ScubaGear/baselines/aad.md

```diff
@@ -649,22 +649,17 @@ Exception cases:
 
 #### MS.AAD.7.6v1 Instructions
 
-1. In **Azure Active Directory** create a new group named **Privileged Escalation Approvers**. This group will contain users that will receive role activation approval requests and approve or deny them.
+1. In **Azure AD Privileged Identity Management (PIM)**, under **Manage**, select **Azure AD roles**.
 
-2. Assign this new group to the Azure AD role **Privileged Role Administrators**. This permission allows users in this group to adjudicate role activation approval requests.
```

Comment:
Can you provide some context for the removal of the dedicated group for approvers?

Reply:
Provided offline and the reviewer approved.